src/emqttd_acl_internal.erl

```diff
@@ -123,10 +105,7 @@ match(Client, Topic, [Rule|Rules]) ->
         {matched, AllowDeny} -> {matched, AllowDeny}
     end.
 
-%%------------------------------------------------------------------------------
 %% @doc Reload ACL
-%% @end
-%%------------------------------------------------------------------------------
 -spec reload_acl(State :: #state{}) -> ok | {error, Reason :: any()}.
```

Comment:
The spec in line 109 uses a record, please define a type for the record and use that instead.